source/extensions/filters/http/ext_authz/ext_authz.cc

```diff
@@ -5,11 +5,22 @@
 #include "common/http/codes.h"
 #include "common/router/config_impl.h"
 
+#include "extensions/filters/http/well_known_names.h"
+
 namespace Envoy {
 namespace Extensions {
 namespace HttpFilters {
 namespace ExtAuthz {
 
+void FilterConfigPerRoute::merge(const FilterConfigPerRoute& other) {
+  disabled_ = other.disabled_;
+  auto begin_it = other.context_extensions_.begin();
+  auto end_it = other.context_extensions_.end();
+  for (auto it = begin_it; it != end_it; ++it) {
```

Comment:
Now I understand what you're getting at; we're merging objects rather than protos.. I had thought this was all at the proto level... 🤕

Reply:
yes, though i think it turned out ok - there are no more copies than needs to be.